main/src/ca/uwaterloo/flix/tools/Tester.scala

```diff
@@ -108,7 +108,7 @@ object Tester {
             case _ => TestResult.Success(sym, "Returned non-boolean value.")
           }
         } catch {
-          case ex: FlixError =>
+          case ex: Exception => // TODO JLS: Maybe delete this and call Main.main() instead or keep
```

Comment:
@magnus-madsen Do you have a comment on how this case should be handled?

Reply:
This is fine actually